### DISCUSSION TITLE
Traccar LXC: with SQL instead of H2-Database

### DISCUSSION THREAD

**Discussion by seneca13** [tteck/Proxmox#3199] [General]
First BIG THNX for Your hard work!

Your traccar LXC uses the default H2 database.
Traccar itself strongly recommends to use the embedded H2 database for testing purposes only:
[By default Traccar uses an embedded H2 database, but we don't recommend for it for production](https://www.traccar.org/mysql/)

 Could You please make an Traccar LXC with a SQL-database?

**Comment by tteck** [comment]
How are you using it for production?

**Reply by seneca13** [reply]
Car fleet of an home nursery service
